### ISSUE TITLE
[Bug][a11y][Docs]: Responsive side-nav and settings overlays should be modal and keyboard accessible

### ISSUE THREAD

**Issue by majornista** [adobe/spectrum-web-components#3428]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

Documentation

### Expected behavior

On documentation pages, when a responsive side-nav or settings overlay is open, the underlying content should be `inert` and the overlay should close when the user presses the Esc key.

### Actual behavior

Content on documentation pages, remains keyboard accessible underneath the side-nav or settings overlay when expanded.

### How can we reproduce this issue?

3. Scroll to '....'
4. Check console
5. See error
